analysis/3_coregister-and-source-recon-bulk.py

```diff
@@ -0,0 +1,240 @@
+"""Coregistration and source reconstruction 
+Created AXL 28/10/24
+
+The script should be first run for all subjects. Then for subjects
+whose coregistration looked a bit off we re-run this script just for that
+particular subject.
+
+This does not use OSL batch utils/config-file setup.
+"""
+
+############## ------- Import packages and define functions needed ---------- ############
+
+import os
+import os.path as op
+import osl
+from pprint import pprint
+from osl import utils
+from osl import source_recon
+import numpy as np
+import coinsmeg_data as coinsmeg
+from IPython.display import HTML, display
+import mne
+from glob import glob
+import pathlib
+
+# functions
+def copy_polhemus_files(recon_dir, subject, preproc_file, smri_file, logger):
```

Comment:
polhemus_dir should be added as a local input argument to the function here, I think - at the moment you are relying on it as being defined as a global variable below.